packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.h

```diff
@@ -181,6 +201,7 @@ typedef void (^RCTHostDidReceiveJSErrorStackBlock)(
 @property (nonatomic, strong, nullable) RCTBridge *bridge;
 @property (nonatomic, strong, nullable) RCTHost *reactHost;
 @property (nonatomic, strong, nullable) RCTSurfacePresenterBridgeAdapter *bridgeAdapter;
+@property (nonatomic, strong, nonnull) RCTRootViewFactoryConfiguration *configuration;
```

Comment:
I'd rather not do it.
The idea of the pattern is that you create the configuration object and you pass it to the RootViewFactory. Changing the configuration after the factory is created exposes it to the mutability problems. Is it possible to keep the same pattern we were using for the other blocks?

RCTAppDelegate declare the method in the public interface. The RCTAppDelegate creates the configuration passing the default method to the RootView. If the app's AppDelegate implements the method, objective-c will automatically use the subclass method instead of the RCTAppDelegate one.